lib/index.js

```diff
@@ -27,10 +38,16 @@ module.exports = {
     container: {
       label: 'Container',
       type: 'text'
+    },
+    azureType: {
+      label: 'Azure Type',
+      type: 'enum',
+      values: [azureTypes.base.label, azureTypes.china.label]
```

Comment:
Why aren't you using Object.values here?

Reply:
Do you prefer something like this?
`Object.values(azureTypes).map(x => x.label)`